src/lib/settings.js

```diff
@@ -25,10 +27,16 @@ export const DEFAULT_SETTINGS = {
     startingRow: 0,
     placeColumn: 0,
     roundsColumns: null,
+    nameColumns: null,
+    nameColumnHeaders: ['name', 'player', 'gracz', 'imię'],
+    nameCellExpression : '(?=^.*[A-Z][a-z]{3,})(?!.*([Kk][yy][uu]|[Dd][Aa][Nn]))',
```

Comment:
RegExp is too complex e.g. `'(?=^.*[a-z]{3,})(?!.*(k|d|p|kyu|dan|pro))'` (not checked).
Remember that you're using it with _ignore-case_ flag. Also add _pro_ level support.